### PR TITLE
add Bricks Builder icon

### LINKED ISSUE #9710: Bricks Builder
### Brand Name

Bricks Builder

### Website

https://bricksbuilder.io/

### Popularity Metric

https://www.similarweb.com/website/bricksbuilder.io/#overview

Global Rank
#185,200

### Official Resources for Icon and Color

https://bricksbuilder.io/wp-content/uploads/2021/02/bricks-logo-complete.svg

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by Mvlprem**
bricks

**Issue:** closes #9710

**Similarweb rank:** [192,471](https://www.similarweb.com/website/bricksbuilder.io/#overview)

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

Official svg can be downloaded from their website [Media kit](https://bricksbuilder.io/bricks-builder-brand-assets.zip)

Svg used ` bricks-favicon-b.svg ` from the downloaded kit. Hex taken from the logo.

### CODE REVIEW

**Comment by adamrusted** [comment]
Thanks for this @Mvlprem!
I think I'd either expect to see some differentiation between the shades of yellow used in the icon or - as used on their website - the `b` letter in `bricks-favicon-b.svg`. What are your thoughts on this?

**Comment by adamrusted** [comment]
That's fine on selecting the hex for us to use - but in terms of the icon, given we're a monochrome icon library I'd expect to either see some gaps between the colours in the icon you selected - or we should use the monochrome icon they provided, being the `b` one I mentioned above :)

**Comment by Mvlprem** [comment]
> That's fine on selecting the hex for us to use - but in terms of the icon, given we're a monochrome icon library I'd expect to either see some gaps between the colours in the icon you selected - or we should use the monochrome icon they provided, being the `b` one I mentioned above :)

Initially i wanted to go with the **`b`** icon too but upon scaling down the icon it becomes `18.73 in width and 24 in height` now these dimensions throw linting error like this ` <path> must be centered at (12, 12); the center is currently (12.367, 12) ` I don't know how to get around this issue as for my understanding it's due to decimal points.

please let me know how to get around this as i was facing this issue with other icons too and i'll update it to **`b`** icon.

**Comment by Mvlprem** [comment]
> Let me take a look at the `b` path. The linter can be a fickle beast - so we might just need to optimise to a higher precision...

Hello @adamrusted I've updated the icon to **`b`** logo please take a look.

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
SVG looks good to me now, @Mvlprem 👍🏻 Just 2 small suggestions for the data and this will be ready to merge.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -2142,6 +2142,11 @@
             "hex": "0B996E",
             "source": "https://www.brevo.com"
         },
+        {
+            "title": "Bricks Builder",
+            "hex": "212121",
```
[the remark this answers is not in the document]
Comment (Mvlprem) [reply _data/simple-icons.json]: Hello @PeterShaggyNoble updated **hex** & **title**

**Review by PeterShaggyNoble — APPROVED** [review]

**Pull request merged**